tests/20-basic-component-check.py

```diff
@@ -22,34 +21,16 @@ class IntegrationTest(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        # Get the relative bundle path from the environment variable.
-        cls.bundle = os.getenv('BUNDLE', 'local.yaml')
-        # Create a path to the bundle based on this file's location.
-        cls.bundle_path = os.path.join(cls.bundle)
-        # Normalize the path to the bundle.
-        cls.bundle_path = os.path.abspath(cls.bundle_path)
-
-        print('Deploying bundle: {0}'.format(cls.bundle_path))
         cls.deployment = amulet.Deployment()
```

Comment:
Nope, this is indeed a little funky. I'm relying on the behavior of bundletester to stand up the bundle, and then to not reset the environment before proceeding to execute the tests.

This is a byproduct of the v4 bundle format and how amulet parses that yaml and what it does with it. We were getting odd test failures due to series mismatch because amulet was not specifying series correctly.

I skipped the middle man, and just stood up the deployment. I dont want amulet to control it so much as i want it to introspect it and tell me its characteristics.